src/pages/docs/actions.mdx

```diff
@@ -76,3 +76,49 @@ You can retrieve a registered UI schema via the `findUISchema` function.
 
 This action allows to unregister a previously registered UI schema
 
+## `findUISchema(uischemas: object[], schema: JsonSchema, schemaPath: string, path: string, fallbackLayoutType?: string, control?: ControlElement, rootSchema?: JsonSchema)`
+
+<ApiLink link='/public/api/core/globals.html#finduischema' title='API' /> 
+
+To retrieve the registered UISchema you can call the 'findUISchema' function which is provided through the properties.
+This function needs the uischemas (which consists of a tester and a uischema), the schema, the schemaPath and a subpath. Optionals parameters are a fallbackLayoutType (`VerticalLayout`by default), a control and a rootSchema.
```

Comment:
Would be nice if you could unify the spelling of uischema to `UI schema` in the whole page to be (mostly) consistent to the rest of the website.